dsl/src/test/java/org/mule/test/config/ast/ArtifactAstValidationsTestCase.java

```diff
@@ -53,6 +84,65 @@
 @Story(DSL_VALIDATION_STORY)
 public class ArtifactAstValidationsTestCase extends AbstractMuleContextTestCase {
 
+  private Injector injector;
+
+  @Rule
+  public SystemProperty systemProperty = new SystemProperty(ENFORCE_EXPRESSION_VALIDATION_PROPERTY, "true");
+
+  class BasicModule extends AbstractModule {
+
+    private Map<org.mule.runtime.api.config.Feature, Boolean> featureBooleanMap =
+        singletonMap(ENFORCE_EXPRESSION_VALIDATION, Boolean.valueOf(true));
+
+    @Override
+    protected void configure() {
+      bind(ArtifactAstValidatorBuilder.class).to(DefaultValidatorBuilder.class);
+      bind(ExtendedExpressionManager.class).to(DefaultExpressionManager.class);
+      bind(ValidationsProvider.class).to(CoreValidationsProvider.class);
+      bind(MuleContext.class).to(DefaultMuleContext.class);
+      bind(TransformersRegistry.class).to(DefaultTransformersRegistry.class);
+      bind(Object.class).to(String.class);
+      bind(Transformer.class).to(StringToBoolean.class);
```

Comment:
are these needed with the methods below?

Reply:
Sorry tried that, test wont run, got errors

```
Requested by:
1  : DefaultExpressionManager.setMuleContext(DefaultExpressionManager.java:379)
      \_ for 1st parameter muleContext
     at DefaultExpressionManager.setMuleContext(DefaultExpressionManager.java:379)
     at ArtifactAstValidationsTestCase$BasicModule.configure(ArtifactAstValidationsTestCase.java:100)

Learn more:
  https://github.com/google/guice/wiki/MISSING_IMPLEMENTATION

2) [Guice/MissingImplementation]: No implementation for TransformersRegistry was bound.

Requested by:
1  : DefaultExpressionManager.setTransformersRegistry(DefaultExpressionManager.java:384)
      \_ for 1st parameter transformersRegistry
     at DefaultExpressionManager.setTransformersRegistry(DefaultExpressionManager.java:384)
     at ArtifactAstValidationsTestCase$BasicModule.configure(ArtifactAstValidationsTestCase.java:100)
```